macros/utils/table_operations/merge_sql.sql

```diff
@@ -19,3 +19,29 @@
     {% do return(macro(target_relation, tmp_relation, unique_key, dest_columns)) %}
     {{ return(merge_sql) }}
 {% endmacro %}
+
+{% macro athena__merge_sql(target_relation, tmp_relation, unique_key, dest_columns, incremental_predicates) %}
```

Comment:
Ideally, this should be done in dbt-athena

Reply:
@artem-garmash currently we have https://github.com/dbt-athena/dbt-athena/blob/main/dbt/include/athena/macros/materializations/models/incremental/merge.sql#L50 - did you have a look? that is not working there?

Reply:
It doesn't seems that we overwrite  `get_merge_sql`, maybe in this case is fine to leave then `athena__merge_sql`here